### ISSUE TITLE
Initial Install SQL Error

### ISSUE THREAD

**Issue by ItsKy1e** [Whisparr/Whisparr#67]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

I installed the latest version, 2.0.0.151 and I am getting a SQL error about a column being false

### Expected Behavior

The webpage should load

### Steps To Reproduce

Fresh install on Ubuntu

### Environment

```markdown
- OS: Ubuntu Linux 18.04.6 LTS
- Whisparr:nightly, today 4/13/2023
- Docker Install:no
- Using Reverse Proxy:no
- Browser:chrome
```

### What branch are you running?

Main

### Trace Logs?

2023-04-13 08:09:34.8|Trace|EventAggregator|HealthCheckCompleteEvent <~ NotificationService
2023-04-13 08:09:34.8|Trace|Http|Res: 3 [GET] /api/v3/localization: 200.OK (246 ms)
2023-04-13 08:09:34.8|Debug|Api|[GET] /api/v3/localization: 200.OK (246 ms)
2023-04-13 08:09:34.8|Trace|Http|Res: 12 [GET] /api/v3/system/status: 200.OK (58 ms)
2023-04-13 08:09:34.8|Debug|Api|[GET] /api/v3/system/status: 200.OK (58 ms)
2023-04-13 08:09:34.6|Error|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|An unhandled exception has occurred while executing the request.

[v2.0.0.151] code = Error (1), message = System.Data.SQLite.SQLiteException (0x800007BF): SQL logic error
no such column: false
   at System.Data.SQLite.SQLite3.Prepare(SQLiteConnection cnn, String strSql, SQLiteStatement previous, UInt32 timeoutMS, String& strRemain)
   at System.Data.SQLite.SQLiteCommand.BuildNextCommand()
   at System.Data.SQLite.SQLiteCommand.GetStatement(Int32 index)
   at System.Data.SQLite.SQLiteDataReader.NextResult()
   at System.Data.SQLite.SQLiteDataReader..ctor(SQLiteCommand cmd, CommandBehavior behave)
   at System.Data.SQLite.SQLiteCommand.ExecuteReader(CommandBehavior behavior)

[whisparr.trace.txt](https://github.com/Whisparr/Whisparr/files/11222549/whisparr.trace.txt)

**Comment by l2noob2** [comment]
I was able to fix mine by moving from ubuntu 18.04.6 lts (which is end of life) to 20.04.6

sudo do-release-upgrade

cheers

**Comment by Qstick** [comment]
Already fixed
